### DISCUSSION TITLE
[How To] Have 4 bottom tabs with drawer on one tab

### DISCUSSION THREAD

**Discussion by launchthatbrand** [expo/router#499] [Q&A]
I have a repo I'm building with bottom tabs set up. Now I want to add a drawer nav on ONE of the tabs (the second tab).

https://github.com/launchthatbrand/t3-turbo-and-clerk/tree/solito-tw

Does anyone have a working example for how to use tabs and drawer together?

I just found this:
https://github.com/Jeroen-G/expo-router-repro/tree/master/app

but not sure if using Links in Drawer is the correct way rather than using <Drawer.Screen

also @EvanBacon can you clarify why there are 2 sections fo the documentation for Drawer?

https://expo.github.io/router/docs/migration/react-navigation/drawer-navigator/
and
https://expo.github.io/router/docs/migration/react-navigation/drawer/

**Comment by wcastand** [comment]
You have to create your Tabs and in the Screen of the second tab make a layout with the Drawer.


```
// app/tabs/_layout

export default function TabLayout(){
return (<Tabs>
<Tabs.Screen name='tab1' />
<Tabs.Screen name='tab2' />
<Tabs.Screen name='tab3' />
<Tabs.Screen name='tab4' />
</Tabs>)
}
```

```
// app/tabs/tab2/_layout

export default function Tab2Layout(){
  return(<Drawer />)
}

```
